webClient/src/app/editor/code-editor/monaco/monaco.config.ts

```diff
@@ -119,7 +119,7 @@ export const DEFAULT_CONFIG: MonacoConfigItem[] = [
     {
       attribute: 'theme',
       type: ConfigItemType.array,
-      values: ['vs', 'vs-dark'],
+      values: ['vs (Beta)', 'vs-dark'],
```

Comment:
What's beta about "vs"? it's a theme name. Can you even successfully change the theme to light after having changed the name?